test/intl402/Temporal/TimeZone/prototype/getNextTransition/subtract-second-and-nanosecond-from-last-transition.js

```diff
@@ -0,0 +1,55 @@
+// Copyright (C) 2022 André Bargull. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+
+/*---
+esid: sec-temporal.timezone.prototype.getnexttransition
+description: >
+  Compute next transition when seconds resp. nanoseconds are subtracted from the last transition.
+features: [Temporal]
+---*/
+
+// From <https://github.com/eggert/tz/blob/main/europe>:
+//
+// # Zone  NAME       STDOFF  RULES FORMAT  [UNTIL]
+// Zone  Europe/Paris 0:09:21 -     LMT     1891 Mar 15  0:01
+//                    0:09:21 -     PMT     1911 Mar 11  0:01 # Paris MT
+
+let tz = new Temporal.TimeZone("Europe/Paris");
+
+let zdt = new Temporal.PlainDateTime(1800, 1, 1).toZonedDateTime(tz);
+assert.sameValue(zdt.toString(), "1800-01-01T00:00:00+00:09[Europe/Paris]");
```

Comment:
FYI, we are usually avoiding using `toString()` to make assertions about Temporal objects, though here I think it is justified for easy comparison to the TZDB excerpt above.